react/README.md

```diff
@@ -470,7 +470,7 @@ We don’t recommend using indexes for keys if the order of items may change.
 
     // good
     <Foo
-      ref={(ref) => { this.myRef = ref; }}
+      ref={ref => this.myRef = ref}
```

Comment:
in fact, we definitely still need it - this guide strictly forbids using assignments in expression position.